README.md

```diff
@@ -1,70 +1,154 @@
-# COVID Safe Paths <img align="left" src="./assets/ShieldKeyHold512x512.png" data-canonical-src="./assets/ShieldKeyHold512x512.png" width="40" height="40"/> ![Android and iOS build on MacOS](https://github.com/tripleblindmarket/covid-safe-paths/workflows/Android%20and%20iOS%20build%20on%20MacOS/badge.svg)
-
-**Applying the technology and philosophy of Private Kit to COVID**
+<div align="center">
+  <h1>COVID Safe Paths</h1>
+
+  <a href="https://covidsafepaths.org">
+    <img
+      height="80"
+      width="80"
+      alt="safe paths logo"
+      src="./assets/Safe_Paths_Logo.png"
+    />
+  </a>
+
+  <p>
+    Applying the technology and philosophy of Private Kit to COVID-19
+  </p>
+  
+  <b>**https://covidsafepaths.org | https://safepaths.mit.edu**</b>
+</div>
+
+<hr />
 
 Help us stop COVID-19.
 
-We’re building the next generation of secure location logging to preserve privacy and #flattenthecurve
+We’re building the next generation of secure location logging to preserve privacy and _#flattenthecurve_
 
 Location logs provide time-stamped records of where users have been, allowing them to share information with health officials accurately and quickly. This helps support contact tracing efforts to slow the spread of the virus.
 
-What’s truly special about Safe Paths, though, is its privacy protection. Data never leaves a user's device without their password entry and explicit consent. The location log generated by Safe Paths cannot be accessed from outside the user’s device, meaning data transfer occurs only if the user chooses to share it with a researcher or health official.
+What’s truly special about Safe Paths, though, is its privacy protection.
 
-----
+Data never leaves a user's device without their password entry and explicit consent. The location log generated by Safe Paths cannot be accessed from outside the user’s device, meaning data transfer occurs only if the user chooses to share it with a researcher or health official.
 
 <img align="right" src="./assets/PreviewUI.png" data-canonical-src="./assets/PreviewUI.png"/>
 
-Safe Paths is a ‘privacy-first’ app that allows you to log your GPS trails on your own phone. The information is stored locally and never shared with anyone (not even with us or MIT) until you explicitly decide to manually export the data. The location log generated by Safe Paths cannot be accessed from outside the user’s device. Location information can be imported and exported by the user and used in other projects and applications.
+## Overview
+
+Safe Paths is a ‘privacy-first’ app that allows you to log your GPS trails on your own phone. The information is stored locally and never shared with anyone (not even with us or MIT) until you explicitly decide to manually export the data.
+
+The location log generated by Safe Paths cannot be accessed from outside the user’s device. Location information can be imported and exported by the user and used in other projects and applications.
```

Comment:
The location log generated by Safe Paths cannot be accessed from outside the user’s device. However, the user can import and export their location information and use it in other projects and applications.